h_periodic/lms_beat.py

```diff
@@ -32,6 +33,11 @@
             "task": "lms.tasks.rsa_key.rotate_keys",
             "schedule": timedelta(hours=1),
         },
+        "send_instructor_email_digests": {
+            "task": "lms.tasks.email_digests.send_instructor_email_digest_tasks",
+            "schedule": crontab(hour=5, minute=15),
```

Comment:
I could imagine a v35 of this dialing with weekends in a different way. For now, if you got annotations on Friday, you'll get them on Saturday, that's fine.

Reply:
Yep. I think we'd actually have to take care of that in the task in the LMS app: if it's Saturday or Sunday then don't do anything. If it's Monday then report on the last 3 days of activity instead of the last 1. Then we could additionally change this cron to not even run the task on Sat or Sun, but that wouldn't strictly be necessary